Sources/armory/renderpath/RenderPathDeferred.hx

```diff
@@ -491,7 +518,7 @@ class RenderPathDeferred {
 		}
 		#end
 
-		#if (rp_ssr_half || rp_ssgi_half)
+		#if (rp_ssr_half || rp_ssgi_half || rp_voxels != "Off")
```

Comment:
I am using half depth for the call to getPos() in the resolve_diffuse/spec/ao shader simply because I don't know how to get the _main depth buffer from